### PR TITLE
bump heapster to v0.11.0 and grafana to v0.7.0

### PR DESCRIPTION
**Pull request by idoshamun**

### CODE REVIEW

File: cluster/addons/cluster-monitoring/v1beta3/influxdb-grafana-controller.yaml [hunk 1]
```diff
@@ -24,10 +24,10 @@ spec:
             - containerPort: 8086
               hostPort: 8086
         - name: grafana
-          image: gcr.io/google_containers/heapster_grafana:v0.6
+          image: gcr.io/google_containers/heapster_grafana:v0.7
           env: 
             - name: "INFLUXDB_EXTERNAL_URL"
-              value: '/api/v1beta1/proxy/services/monitoring-grafana/db/'
+              value: '/api/v1beta3/proxy/namespaces/system/services/monitoring-grafana/db/'
```
Comment (vishh) [inline cluster/addons/cluster-monitoring/v1beta3/influxdb-grafana-controller.yaml]: Have you tested this change? If you were to make this change on the heapster repo [here](https://github.com/GoogleCloudPlatform/heapster/blob/master/deploy/kube-config/influxdb/v1beta3/influxdb-grafana-controller.json#L41), it will get tested automatically by CI!

**Comment by jlowdermilk** [comment]
Also need to make sure the updated image version is pushed to gcr.io

**Comment by vishh** [comment]
I did push the images to gcr.io. So that should not be an issue!

File: cluster/addons/cluster-monitoring/v1beta3/influxdb-grafana-controller.yaml (hunk 1 above)
Comment (idosh) [inline cluster/addons/cluster-monitoring/v1beta3/influxdb-grafana-controller.yaml]: Actually it might be a good idea to revert this change.
I made it especially for my cluster.
And everything is tested of course.

**Comment by vishh** [comment]
LGTM
